src/window.rs

```diff
@@ -369,7 +369,7 @@ impl Window {
         self.window().hide();
     }
 
-    fn window(&self) -> &glutin::Window {
+    pub fn window(&self) -> &glutin::Window {
```

Comment:
In order to toggle fullscreen, we needed access to the `glutin::Window` so I had to make this method `pub`.

I'm not sure if that's okay? We can always make it `pub(crate)` instead if that's preferred.